src/list/manage/containers/connected-promote-banner.js

```diff
@@ -9,18 +9,20 @@ import React from "react";
 import { connect } from "react-redux";
 
 import { PromoteDeviceBanner, PromoteFxABanner } from "../components/promote-banner";
-import { openHomepage, openSyncPrefs } from "../../actions";
+import { openSyncPrefs, openAppStore, openAndroidStore } from "../../actions";
 
-export function CurrentPromotionBanner({hasProfile, onClickDeviceBanner, onClickFxABanner}) {
+export function CurrentPromotionBanner({hasProfile, onAppStoreClick, onAndroidStoreClick, onClickFxABanner}) {
```

Comment:
this is a 🚲🏠 nit: I prefer keeping symmetry where possible.  To me, (`AppStore`, `PlayStore`) or (`AppleStore`, `GoogleStore`) or (`iosStore`, `AndroidStore`) are symmetrical.  `AppStore` and `AndroidStore` are not symmetrical to me.

Since the actions are `openAppStore` and `openPlayStore`, would that also make sense here?

Reply:
Yeah, I wasn't happy with those names either. I'll roll with `openAppStore` and `openPlayStore`.